### PR TITLE
Migration content update

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by msewaweru** [comment]
Hi @f4rty - this is ready for your review.

**Comment by msewaweru** [comment]
@f4rty, please review this. I have incorporated feedback from our discussions. Thanks.

**Comment by f4rty** [comment]
Reviewed and approved

**Comment by msewaweru** [comment]
@adamedx , requesting for your review of the updated migration content. Thanks.

File: microsoftgraph/docs-conceptual/migration-steps.md [hunk 1]
```diff
@@ -3,57 +3,89 @@ title: "Migrate from Azure AD PowerShell to the Microsoft Graph PowerShell SDK."
...
 
-- There is not yet an equivalent of -SearchString for Get-AzureADUser, Get-AzureADGroup commands. Use -Filter instead. For example, `Get-MgUser -Filter "DisplayName eq 'Lee Gu'"` returns the user whose display name is equal to the specified string.
-- Search does not yet work for any Azure AD commands.
-- You need to use hash tables to pass nested parameters. Here is a sample of [Nested parameters](https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/samples/9-Applications.ps1#L28-L43).
-- **Pro-tip**: Use the Microsoft Graph PowerShell ConsistencyLevel parameter. It lets you do $count! This is best used for read-only scenarios and you need to be more careful when making changes. To learn more about the ConsistencyLevel parameter see [Advanced query parameters](/graph/aad-advanced-queries).
+- There isn't yet an equivalent of **-SearchString** for `Get-AzureADUser` and `Get-AzureADGroup` commands. Use **-Filter** instead. For example, `Get-MgUser -Filter "DisplayName eq 'Lee Gu'"` returns the user whose display name is equal to the specified string.
+- Search doesn't yet work for any Azure AD commands.
```
Comment (adamedx) [inline microsoftgraph/docs-conceptual/migration-steps.md:83]: This is not true -- Search works for many commands such as `Get-MgUser` -- as long as you use `ConsistencyLevel` of `Eventual`. The behavior is not the same as `SearchString` but arguably better.

**Review by adamedx — CHANGES_REQUESTED** [review]
This is a good start, though I think we may need something more in-depth soon.

**Comment by msewaweru** [comment]
@timayabi2020, please merge this.

**Review by timayabi2020 — APPROVED** [review]

**Pull request merged**